pyphare/pyphare/pharein/diagnostics.py

```
@@ -81,6 +84,26 @@ def validate_timestamps(clazz, key, **kwargs):
     return timestamps
 
 
+def validate_elapsed_timestamps(clazz, key, **kwargs):
```

Ruff (0.14.11)

Unused function argument: `clazz`

(ARG001)